dll/win32/profmap/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+
+spec2def(profmap.dll profmap.spec ADD_IMPORTLIB)
```

Comment:
Unless we import from this dll, we do not need `ADD_IMPORTLIB`

Reply:
Removed. Thanks.